pydarn/plotting/fan.py

```diff
@@ -207,20 +210,34 @@ def plot_fan(cls, dmap_data: List[dict], ax=None, ranges: List = [],
             rsep = dmap_data[0]['rsep']
         except KeyError:
             rsep = 45
+
+        if coords != Coords.GEOGRAPHIC and projs == Projs.GEO:
+            raise plot_exceptions.NotImplemented("AACGM coordinates currently"\
+                                                 " is not implemented right"\
+                                                 " now, if you would like to"\
+                                                 " see it sooner please help"\
+                                                 " out on "\
+                                                 "https://github.com"\
+                                                 "/SuperDARN/pyDARN")
```

Comment:
```suggestion
            raise plot_exceptions.NotImplemented("AACGM coordinates are"\
                                                 " not implemented for "\
                                                 " geographic projections right"\
                                                 " now, if you would like to"\
                                                 " see it sooner please help"\
                                                 " out at "\
                                                 "https://github.com"\
                                                 "/SuperDARN/pyDARN")
```

I'm liking the subtle recruiting of devs :P